### PR TITLE
[ROCKETMQ-291]Fix System.out.printf throws UnknownFormatConversionException

### PR DESCRIPTION
**Pull request by lindzh**
###  What is the purpose of the change 
When using mqadmin command printmsg to print retry topic msgs,it throw an exception like this:
com.alibaba.rocketmq.client.exception.MQClientException: Can not find Message Queue for this 
```
topic, %%RETRY%%CID_MERSS_CONSUOUP_UAT
See http://rocketmq.apache.org/docs/faq/ for further details.
  at com.alibaba.rocketmq.client.impl.MQAdminImpl.fetchSubscribeMessageQueues(MQAdminImpl.java:161)
  at com.alibaba.rocketmq.client.impl.consumer.DefaultMQPullConsumerImpl.fetchSubscribeMessageQueues(DefaultMQPullConsumerImpl.java:136)
  at com.alibaba.rocketmq.client.consumer.DefaultMQPullConsumer.fetchSubscribeMessageQueues(DefaultMQPullConsumer.java:225)
  at com.alibaba.rocketmq.tools.command.message.PrintMessageSubCommand.execute(PrintMessageSubCommand.java:120)
  at com.alibaba.rocketmq.tools.command.MQAdminStartup.main0(MQAdminStartup.java:128)
,. JIRA: https://issues.apache.org/jira/browse/ROCKETMQ-278.
```
jira:https://issues.apache.org/jira/browse/ROCKETMQ-291
### Brief change log
* fix all system.out.prinf using in rocketmq.

### Verifying this change
* This change is a trivial rework / code cleanup with unitout test coverage.
* execute mqadmin printmsg -t %RETRY%CID_MERSS_CONSUOUP_UAT.

Follow this checklist to help us incorporate your contribution quickly and easily:
- [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/ROCKETMQ/issues/)  filed for the change (usually before you start working on it). Trivial changes like typos do not require a JIRA issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x]  Format the pull request title like [ROCKETMQ-XXX] Fix UnknownException when host config not exist. Each commit in the pull request should have a meaningful subject line and body.
- [x]  Write a pull request description that is detailed enough to understand the pull request :
  - What is the purpose of the change
  - Brief change log
  - Verifying this change
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-rocketmq/tree/master/test).
- [x]  Run mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle to make sure basic checks pass. Run mvn clean install -DskipITs to make sure unit-test pass. Run mvn clean test-compile failsafe:integration-test to make sure integration-test pass. 
- [x]  If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: tools/src/main/java/org/apache/rocketmq/tools/command/topic/UpdateTopicSubCommand.java [hunk 1]
```diff
@@ -169,7 +169,7 @@ public void execute(final CommandLine commandLine, final Options options,
                     }
                     defaultMQAdminExt.createOrUpdateOrderConf(topicConfig.getTopicName(),
                         orderConf.toString(), true);
-                    System.out.printf(String.format("set cluster orderConf. isOrder=%s, orderConf=[%s]",
+                    System.out.printf("%s", String.format("set cluster orderConf. isOrder=%s, orderConf=[%s]",
```
Comment (zhouxinyu) [inline tools/src/main/java/org/apache/rocketmq/tools/command/topic/UpdateTopicSubCommand.java]: Remove inner `String.format` will be more clear

File: tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerSubCommand.java [hunk 2]
```diff
@@ -114,7 +114,7 @@ public void execute(CommandLine commandLine, Options options, RPCHook rpcHook) t
                             String result =
                                 ConsumerRunningInfo.analyzeProcessQueue(next.getKey(), next.getValue());
                             if (result.length() > 0) {
-                                System.out.printf(result);
+                                System.out.printf("%s", result);
```
Comment (zhouxinyu) [inline tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerSubCommand.java]: If no need format, just `System.out.println`?

File: tools/src/main/java/org/apache/rocketmq/tools/command/topic/UpdateTopicSubCommand.java (hunk 1 above)
Comment (lindzh) [reply tools/src/main/java/org/apache/rocketmq/tools/command/topic/UpdateTopicSubCommand.java]: Right,and this problem has been fixed.

File: tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerSubCommand.java (hunk 2 above)
Comment (lindzh) [reply tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerSubCommand.java]: Yes,and this problem has been fixed.

Comment (zhouxinyu) [reply tools/src/main/java/org/apache/rocketmq/tools/command/consumer/ConsumerSubCommand.java]: OK

**Comment by zhouxinyu** [comment]
LGTM now~

**Comment by dongeforever** [comment]
LGTM

**Comment by dongeforever** [comment]
@lindzh This PR has been merged.

**Pull request closed without merging**